main.tf

```diff
@@ -12,27 +12,38 @@ locals {
 }
 
 module "task_label" {
-  source     = "cloudposse/label/null"
-  version    = "0.25.0"
-  enabled    = local.create_task_role
-  attributes = ["task"]
+  source      = "cloudposse/label/null"
+  version     = "0.25.0"
+  enabled     = local.create_task_role
+  attributes  = ["task"]
+  label_order = var.label_order_custom.iam
 
   context = module.this.context
 }
 
 module "service_label" {
-  source     = "cloudposse/label/null"
-  version    = "0.25.0"
-  attributes = ["service"]
+  source      = "cloudposse/label/null"
+  version     = "0.25.0"
+  attributes  = ["service"]
+  label_order = var.label_order_custom.service
```

Comment:
```suggestion
  label_order = var.label_orders.iam
```

Reply:
Doesn't work because its taken for the security group as well

Reply:
We could only change the label stuff even more... but I'm not sure about that. Again because of source reasons.

Reply:
How about we simply add another label module for vpc resources like security groups? Then we can keep what we decided in the meeting and it should remain clean. It's a feature in the end and shouldn't be hard to maintain if needed